content_loader.py

```diff
@@ -0,0 +1,85 @@
+import logging
+import requests
+
+from abc import ABC, abstractmethod
+from typing import Any, Tuple
+
+import credentials
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+
+class ContentLoader(ABC):
+    @abstractmethod
+    def load(self) -> Any:
+        """
+        Abstract method to load content.
+        """
+        raise NotImplementedError("Subclasses must implement this method")
```

Comment:
There is no need to raise this error manually. If necessary, ABC will do it for you.